### PR TITLE
[DE] Add `Hype`

### PR DESCRIPTION
**Pull request by FreezyLemon**
Should be merged after #10317.

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Hype/de.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - train
+---
+
+# Hype
+
+Jede [ausstehende](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) [Beatmap](/wiki/Beatmap) hat einen **Hype-Train**, der grob zeigt, wie viele Spieler am [Ranking](/wiki/Beatmap/Category#ranked) der Beatmap interessiert sind. Nutzer können spezielle [Beatmapdiskussionen](/wiki/Beatmap_discussion) im Tab `Generell (alle Schwierigkeitsstufen)` erstellen und die Option `Hype!` verwenden, um zum Hype-Train beizutragen.
```
Comment (Deimosu) [inline wiki/Beatmap/Hype/de.md:9]:
```suggestion
Jede [ausstehende](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) [Beatmap](/wiki/Beatmap) hat einen **Hype-Train**, der grob zeigt, wie viele Spieler daran interessiert sind, die Map als [Ranked](/wiki/Beatmap/Category#ranked) zu sehen. Nutzer können spezielle [Beatmap-Diskussionen](/wiki/Beatmap_discussion) im Tab `Generell (alle Schwierigkeitsstufen)` erstellen und die Option `Hype!` verwenden, um zum Hype-Train beizutragen.
```

File: wiki/Beatmap/Hype/de.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Ranking-Voraussetzung
+
+Beatmaps müssen mindestens 5 Hypes (eine ganze Leiste an Hype-Train) sammeln, um für eine [Nominierung](/wiki/Beatmap_ranking_procedure#nominations) infrage zu kommen.
```
Comment (Deimosu) [inline wiki/Beatmap/Hype/de.md:13]:
```suggestion
Beatmaps benötigen mindestens 5 Hypes (eine vollständige Leiste des Hype-Trains), um für eine [Nominierung](/wiki/Beatmap_ranking_procedure#nominations) infrage zu kommen.
```

File: wiki/Beatmap/Hype/de.md (hunk 1 above)
Comment (FreezyLemon) [reply wiki/Beatmap/Hype/de.md:9]: Sticking this close to the English wording hurts it a bit IMO. 
How about:

```suggestion
Jede [ausstehende](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) [Beatmap](/wiki/Beatmap) hat einen **Hype-Train**, der grob zeigt, wie viele Spieler daran interessiert sind, dass die Beatmap [gerankt](/wiki/Beatmap/Category#ranked) wird. Nutzer können spezielle [Beatmap-Diskussionen](/wiki/Beatmap_discussion) im Tab `Generell (alle Schwierigkeitsstufen)` erstellen und die Option `Hype!` verwenden, um zum Hype-Train beizutragen.
```

Comment (Deimosu) [reply wiki/Beatmap/Hype/de.md:9]: sounds good to me

**Review by Deimosu — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**